homeassistant/components/device_tracker/unifi.py

```diff
@@ -33,55 +40,87 @@
     vol.Required(CONF_USERNAME): cv.string,
     vol.Required(CONF_PORT, default=DEFAULT_PORT): cv.port,
     vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
+    vol.Optional(CONF_DETECTION_TIME, default=DEFAULT_DETECTION_TIME): vol.All(
+        cv.time_period, cv.positive_timedelta),
+    vol.Optional(CONF_API_VERSION, default=DEFAULT_API_VERSION): cv.string
 })
 
 
-def get_scanner(hass, config):
+def get_scanner(hass, config, ectrl=None):
```

Comment:
This is a standardized method from the Device Tracker component. You should not add extra arguments.

Reply:
If this is used for testing, use Python mocks in your tests to stub it out.